### PR TITLE
Remove NextUpActivity from displaying after intro videos or trailers

### PR DESCRIPTION
**Pull request by MrChip53**
**Changes**
Removes the next up activity from showing up after the intro and trailer videos.

This works by setting intro videos manually to the trailer type and immediately playing the next item in the queue if the current item is a trailer.

This is the default behavior of v0.11

**Issues**

### CODE REVIEW

File: app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java [hunk 1]
```diff
@@ -1048,7 +1048,9 @@ private void itemComplete() {
         if (nextItem != null) {
             Timber.d("Moving to next queue item. Index: " + (mCurrentIndex + 1));
 
-            if (userPreferences.getValue().get(UserPreferences.Companion.getNextUpEnabled())) {
+            BaseItemDto curItem = getCurrentlyPlayingItem();
```
Comment (thornbill) [inline app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java:1051]: It looks like `getCurrentlyPlayingItem()` can return `null` so we need to handle that properly.

File: app/src/main/java/org/jellyfin/androidtv/util/apiclient/PlaybackHelper.java [hunk 2]
```diff
@@ -217,7 +217,11 @@ public void onResponse(ChannelInfoDto response) {
                         @Override
                         public void onResponse(ItemsResult response) {
                             if (response.getTotalRecordCount() > 0){
-                                Collections.addAll(items, response.getItems());
+
+                                for(BaseItemDto intro : response.getItems()){
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/util/apiclient/PlaybackHelper.java:221]:
```suggestion
                                for (BaseItemDto intro : response.getItems()) {
```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java (hunk 1 above)
Comment (MrChip53) [reply app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java:1051]: I believe this is now resolved.

**Review by nielsvanvelzen — APPROVED** [review]
Linter task failure is unrelated

**Review by thornbill — APPROVED** [review]

**Pull request merged**